src/components/Button.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import styled from 'styled-components'
+import buttonBG from '../assets/hc_button.svg'
+import { hexToRgba } from '../utility/utilities'
+
+const buttonWidth = {
+  small: '40px',
+  default: '60px',
+  large: '300px',
+  flex: '',
+}
+
+const buttonHeightPadding = {
+  short: '0.25em',
+  default: '0.5em',
+  tall: '1em',
+}
+
+const StyledButton = styled.a`
+  display: inline-block;
+  text-decoration: none;
+  font-family: ${p => p.theme};
+  font-weight: ${p => p.theme.typography.h2.weight};
+  border: transparent;
+  transition: all 250ms;
+  width: ${p => (p.width ? buttonWidth[p.width] : buttonWidth['default'])};
+  text-align: center;
+  padding: ${p => (p.height ? buttonHeightPadding[p.height] : buttonHeightPadding['default'])}
+    0.75em;
+  border-radius: 3px;
+  margin: 1em;
```

Comment:
I think this can be decided by user? And left out?

Reply:
do you mean the margin?

Reply:
Yea maybe remove margin on the default buttons

Reply:
the padding around the text here makes the button height (also helps the text inside stay vertically centered)

Reply:
![image](https://user-images.githubusercontent.com/47487758/100293444-a0f4bd80-2f38-11eb-90b5-a1af6c350087.png)
lmao

Reply:
all g i added margin to those but removed it from Button.js